package.json

```diff
@@ -15,14 +15,15 @@
     "d3-tip": "^0.9.1",
     "dayjs": "^1.8.8",
     "echarts": "^4.1.0",
+    "element-ui": "^2.12.0",
```

Comment:
Is this necessary? If it is not necessary, it is recommended not to use it.

Reply:
It can be used for quick and easy development, because we don't have ready-made drawer components, and the table function is not perfect.

Reply:
![image](https://user-images.githubusercontent.com/19775780/68353454-9ce6d980-0144-11ea-9e1a-d0098c4d2048.png)
This component is the drawer, element  babel-plugin-import can not avoid hedging of some basic components.  Web components package are for fast developing but not for suitable developing.